web/docs/tql2/operators/read_pcap.md

```diff
@@ -0,0 +1,67 @@
+# read_pcap
+
+Reads and writes raw network packets in [PCAP][pcap-rfc] file format.
+
+[pcap-rfc]: https://datatracker.ietf.org/doc/id/draft-gharris-opsawg-pcap-00.html
+
+```
+read_pcap [emit_file_headers=bool]
+```
+
+## Description
+
+The `read_pcap` operator converts raw bytes representing a [PCAP][pcap-rfc] file into
+events.
+
+[pcapng-rfc]: https://www.ietf.org/archive/id/draft-tuexen-opsawg-pcapng-05.html
+
+:::note PCAPNG
+The current implementation does *not* support [PCAPNG][pcapng-rfc]. Please
+[reach out](/discord) if you would like to see support.
+:::
+
+The structured representation of packets has the `pcap.packet` schema:
+
+```yaml
```

Comment:
Change this to a table like https://docs.tenzir.com/tql2/operators/metrics

and change the type of `data` to `blob`.